README.md

```diff
@@ -186,6 +186,12 @@ ReactDOM.render(<Menu>
             <th>() => document.body</th>
             <td>Where to render the DOM node of popup menu when the mode is horizontal or vertical</td>
         </tr>
+        <tr>
+            <td>popupPlacements</td>
```

Comment:
How about `builtinPlacements`? To be consistent with [`rc-trigger`](https://github.com/react-component/trigger#props)

Reply:
Done 👍🏻